packages/bitcore-payid/test/integration/index.spec.ts

```diff
@@ -118,6 +215,133 @@ describe('PayId', () => {
         expect(verified).be.true;
       });
     });
+  });
+
+  describe('sign & verify', () => {
+    it('should work with Bitcore HD key', () => {
+      const signed = PayId.sign(payId, addressBTC.addressDetails.address, 'BTC', keys.bitcoreHD.toString());
+      expect(signed).to.exist;
+
+      const verified = PayId.verify(payId, signed);
+      expect(verified).be.true;
+    });
+
+    it('should work with Bitcore non-HD key', () => {
+      const signed = PayId.sign(payId, addressBTC.addressDetails.address, 'BTC', keys.bitcore.toString());
+      expect(signed).to.exist;
+
+      const verified = PayId.verify(payId, signed);
+      expect(verified).be.true;
```

Comment:
what about adding a tests to PayId.verify with a different payId => verify should by false; with a different signature => verify should be false.

Reply:
Good idea

Reply:
Done